tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -4,21 +4,37 @@ exports[`assignment-pattern.js 1`] = `
 "const { a /* comment */ = 1 } = b;
 
 const { c = 1 /* comment */ } = d;
+
+let {a //comment
+= b} = c
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 const { a /* comment */ = 1 } = b;
 
 const { c = 1 /* comment */ } = d;
+
+let {
+  //comment
```

Comment:
I think we are already doing something like that somewhere else, i.e. putting the comment upfront.